1905-07-17.xml

```diff
@@ -1,1018 +1,2590 @@
-                            the <date when="1905-07-25">25th inst.</date></p>
-                        <p>N. E. TAMVACO Alexandria agents <measure type="indexNo">23186-20-3-3</measure></p>
-                    </div>
-                    <div type="advert" xml:id="deg-ad-ell01">
-                        <head>Ellerman Lines, Limited.</head>
-                        <table rows="3" cols="6">
-                            <row>
-                                <cell cols="3">CITY LINE to MALTA, LONDON, COLOMBO &amp; CALCUTTA.</cell>
-                                <cell cols="3">
-                                    <p>CITY &amp; HALL LINES. Joint Service to MARSEILLES, LIVERPOOL, BOMBAY
-                                        &amp; KARACHI.</p>
-                                </cell>
-                            </row>
-                            <row>
-                                <cell cols="6">The undermentioned First Class Passenger Steamers will be
-                                    dispatched from Port Said on or about the following dates for</cell>
-                            </row>
-                            <row>
-                                <cell>Malta and London</cell>
-                                <cell><name>S.S. City of Corinth</name></cell>
-                                <cell><date when="1905-07-26">July 26</date></cell>
-                                <cell>Marseilles and Liverpool</cell>
-                                <cell/>
-                                <cell/>
-                            </row>
-                            <row>
-                                <cell>Colombo and Calcutta</cell>
-                                <cell><name>S.S. City of Manchester</name></cell>
-                                <cell><date when="1905-07-12">July 12</date></cell>
-                                <cell>Bombay</cell>
-                                <cell><name>S.S. Anton Hall</name></cell>
-                                <cell><date when="1905-07-13">July 13</date></cell>
-                            </row>
-                        </table>
-                        <p>SALOON FARES:—Port Said to Malta £4.10.0. Marseilles. £8.0.0. London or
-                            Liverpool, £l2.l0.0. Colombo, Calcutta, Bombay or Karachi, £35.0.0. Special
-                            rates for steamers not carrying Doctor or Stewardess. For further particulars
-                            apply to</p>
-                        <p>CORY BROS. &amp; Co., Ltd., Agents for CITY Line, Port Said: W. STAPLEDON &amp;
-                            SON, Agents for Hall Line, Port Said ; or COOK &amp; SON (Egypt), Ltd., Cairo.
-                            <measure type="indexNo">23788-28-8-905</measure></p>
-                    </div>
-                    <div type="advert" xml:id="deg-ad-ngi01" xml:lang="fr">
-   
```

Comment:
You could remove this whole section division. The items in this column are individual.